js/viewTranslations/viewTranslationsModule.js

```diff
@@ -0,0 +1,3 @@
+(function(){
+    angular.module('viewTranlationsModule', []);
```

Comment:
Translations it's very confusing name for this feature.
Please rename it in Broadcast in the whole app